core/src/main/java/org/apache/iceberg/BaseSnapshot.java

```diff
@@ -78,6 +91,19 @@
     this.operation = operation;
     this.summary = summary;
     this.manifestListLocation = manifestList;
+    this.tableLocation = tableLocation;
+    this.shouldUseRelativePaths = shouldUseRelativePaths;
```

Comment:
Nit: was there a reason its 'shouldUseRelativePaths' vs 'useRelativePaths'?  Save a bit of ink ?